Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 
 Compiler Features:
+* Generate warnings on isoltest when Gas Cost Expectations are not being enforced.
```

Comment:
This only concerns the tests and thus should not have a changelog entry.

Reply:
Removed in the latest push